config/config.go

```diff
@@ -107,6 +109,24 @@ func NewConfigData(adapterName string, data []byte) (Configer, error) {
 	return adapter.ParseData(data)
 }
 
+const envKeySign = "$ENV_"
+
+// Getenv return environment variable if env has prefix "$ENV_".
+func Getenv(env interface{}) (string, bool) {
+	if env == nil {
+		return "", false
+	}
+
+	// Onley support string key.
+	if key, ok := env.(string); ok {
+		if len(key) > len(envKeySign) && strings.HasPrefix(key, envKeySign) {
```

Comment:
you can remove `len(key) > len(envKeySign)`, because `strings.HasPrefix` already has that logic